app/assets/javascripts/initializers/initializeAllFollowButts.js

```diff
@@ -1,19 +1,21 @@
+/* global showModal */
```

Comment:
Aside from this eslint comment, the remainder of the changes in this file are all prettier code formatting.